### DISCUSSION TITLE
Improvements to the web UI

### DISCUSSION THREAD

**Discussion by roidelapluie** [cortexproject/cortex#3588] [Ideas]
I'd like to know if something is planned for the cortex UI.

I am not sure we need something complex, but some improvements would definitely help, as currently it is not really welcoming.

**Comment by pracucci** [comment]
It's not something we discussed so far, but I'm very open to it. What are your thoughts? Is it just about styling or features too?

**Comment by roidelapluie** [comment]
Regarding features, it can be annoying when debugging to adapt tenant by hand, for some applications. I wonder if no tenant is passed, for some services, we could have a "landing page" with a dropdown to select the tenant.

Styling would be to add some colors, and a menu (going back to main page is a bit annoying).
